Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.9
+
+WORKDIR /app
+
+COPY . /app
+
+RUN pip install -r /app/requirements.txt
+
+COPY ./dotalytics_api /app/dotalytics_api
```

Comment:
isn't this a dup of L5?

Reply:
if the WORKDIR is set to /app do you need to specify /app everywhere?

Reply:
Yes, you're right. Whoops

Reply:
I removed the /app specification, thanks for the review!